common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -533,4 +533,14 @@ trait FeatureSwitches {
     sellByDate = Some(LocalDate.of(2024, 6, 5)),
     exposeClientSide = true,
   )
+
+  val CrosswordMobileBanner = Switch(
```

Comment:
What's the purpose of this switch if we already have an AB test? https://github.com/guardian/frontend/pull/26867